README.md

```diff
@@ -39,7 +39,7 @@ To build the DAOS VOL connector, the following libraries are required:
 
 + `libhdf5` - The [HDF5](https://www.hdfgroup.org/downloads/hdf5/) library.
             Minimum version required is 1.13.3, compiled with
-            support for both parallel I/O and map objects 
+            support for both [MPICH](https://www.mpich.org) parallel I/O and map objects 
```

Comment:
We shouldn't mention a specific MPI implementation here since the connector will operate fine with implementations other than MPICH.

Reply:
I attended DAOS tutorial today and @mchaarawi said MPICH is necessary.

Reply:
I only use OpenMPI with the DAOS VOL connector, so no, MPICH isn't necessary. It's necessary currently for the tests to run via CTest, but it's not necessary in general. It's fine to document that MPICH is needed for the tests and/or to create an issue explaining that, but I don't think this is the correct place for that.

Reply:
Then, where is a good place to add this information for user?

Reply:
Ideally, we should just fix the issue so the tests aren't restricted to a particular MPI implementation.

Reply:
This should say HDF5 1.14.0 instead of 1.13.3. I don't want to make it sound like that's a legit, released version of the library.

Reply:
probably a misunderstanding. mpich is not necessary for the DAOS VOL. 
mpich is necessary if you want to use the DAOS ADIO driver (MPIIO VFD - native VOL). there is a difference.

Reply:
@mchaarawi , thank you for the clarification. 
I understood what you meant.
My point was: MPICH can work well for everything in DAOS stack so why should user bother with OpenMPI?
Is there any extra benefit of using OpenMPI + HDF5 DAOS VOL?